libraries/standard/mqtt/src/mqtt_lightweight.c

```diff
@@ -123,14 +129,37 @@
  *
  * This value is greater than what is allowed by the MQTT specification.
  */
-#define MQTT_REMAINING_LENGTH_INVALID               ( ( size_t ) 268435456 )
+#define MQTT_REMAINING_LENGTH_INVALID             ( ( size_t ) 268435456 )
 
 /**
  * @brief The minimum remaining length for a QoS 0 PUBLISH.
  *
  * Includes two bytes for topic name length and one byte for topic name.
  */
-#define MQTT_MIN_PUBLISH_REMAINING_LENGTH_QOS0      ( 3U )
+#define MQTT_MIN_PUBLISH_REMAINING_LENGTH_QOS0    ( 3U )
+
+/*-----------------------------------------------------------*/
+
+/* MQTT Subscription packet types. */
+typedef enum MQTTSubscriptionType
+{
+    MQTT_SUBSCRIBE,
+    MQTT_UNSUBSCRIBE
+} MQTTSubscriptionType_t;
+
+/*-----------------------------------------------------------*/
+
+static bool subscriptionPacketSize( const MQTTSubscribeInfo_t * pSubscriptionList,
```

Comment:
We should not use `bool` here. Why not use `MQTTStatus_t`? Also function name can better be `calculateSubscriptionPacketSize`.